### DISCUSSION TITLE
What is the good practice for PostgreSQL cluster storage?

### DISCUSSION THREAD

**Discussion by antonywhatever** [ory/hydra#3519] [Q&A] [answered]
As far I understand dsn with multiple hosts e.g. `postgres://hydra_user:hydra_pwd@hydradb01:5432,hydradb02:5432,hydradb03:5432/hydra` is not supported. What is the best production ready alternative?

**Comment by vinckr** [comment, marked as answer]
Hello @antonywhatever 
Can you share a bit what you are trying to achieve? 
You can connect 1 instance of Hydra to 1 database generally. 
Ory publishes Helm charts for Kubernetes if you need to manage many Ory Hydra deployments: https://github.com/ory/k8s and also offers a [managed service](https://www.ory.sh/network/) with multi-tenancy out of the box.
